casper/src/test/scala/coop/rchain/casper/batch1/MultiParentCasperCommunicationSpec.scala

```diff
@@ -14,55 +15,55 @@ import org.scalatest.{FlatSpec, Inspectors, Matchers}
 // casper and communication has little to do with each other, so commented out
```

Comment:
Is this comment still relevant?

Reply:
No, removed https://github.com/rchain/rchain/pull/3355/commits/00eb5badd57112a8e68e41498402476caf77db2a